### DISCUSSION TITLE
[FeatureRequest]Fit on "BRIO" Rails

### DISCUSSION THREAD

**Discussion by csuno** [StuckAtPrototype/Racer#9] [Ideas]
did you think about, make it fit the Brio rail system? You already showed one in your video!

https://www.brio.us/en-US/products/railway-toys

**Comment by StuckAtPrototype** [comment]
unfortunately it doesn't but, i wish it did! the motors are a little too large, the enclosure could be modified to offset it and then it could be adjusted to fit! the pcb would probably have to be slightly modified

**Comment by csuno** [comment]
than i will edit this topic to a "feature request" ;)
... if its ok!?

**Reply by StuckAtPrototype** [reply]
sure, you can edit the topic to a feature request 👍
